group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/SubscribedTopicMetadataTest.java

```diff
@@ -43,27 +45,62 @@ public void testAttribute() {
                 new TopicMetadata(topicId, topicName, 5, partitionRacks)
             );
         }
-        assertEquals(topicMetadataMap, new SubscribedTopicMetadata(topicMetadataMap).topicMetadata());
+         subscribedTopicMetadata = new SubscribedTopicMetadata(topicMetadataMap);
+    }
+
+
+    @Test
+    public void testAttribute() {
+        setup();
```

Comment:
Instead of doing this, we could use the  `@BeforeEach` annotation. You will find examples in the code base.

Reply:
thankss